cmd/applyconfig/applyconfig.go

```diff
@@ -211,7 +211,7 @@ func applyConfig(rootDir string, o *options) error {
 		// oc-cli works with the symlinks targeting files
 		// here we need to handle symlinks targeting folders recursively
 		if info.Mode()&os.ModeSymlink != 0 {
-			target, err := os.Readlink(path)
+			target, err := filepath.EvalSymlinks(path)
 			if err != nil {
 				logrus.WithError(err).Errorf("failed to readlink: %s", path)
```

Comment:
Shouldn't we also return here if there was an error? Otherwise the `os.Stat` will panic

Reply:
Forgot to do this.
Updated.